extensions/music/package.json

```diff
@@ -17,90 +17,85 @@
     "fix-lint": "ray lint --fix",
     "lint": "ray lint"
   },
-  "commands": [
```

Comment:
Sounds good, I will add back most of them but I do feel like some of them are never used, such as rate track or dislike track. What do you think? Do you use these ever?

Reply:
I personally don't use them but I implemented because some users asked for it 😅
Since it seems that the extension has 9k installs it's hard without any analytics to understand which one are used most...

Reply:
Sounds good, i'll add it back